ansible/roles/products/teleop/server/desktop-icons/tasks/only-simulation.yml

```diff
@@ -189,7 +189,7 @@
     desktop_icon_path: "Shadow Advanced Launchers/3 - Launch Right Teleop Simulation 8DOF"
     project_name_input: "{{ project_name_input_value }}"
     launch_file_input: "{{ launch_file_input_gui_right_8DOF }}"
-  when: arm_servo|bool == true
+  when: arm_servo|bool
```

Comment:
Does this have to be defined in the one-liner or is it optional?

Reply:
Its optional, false by default, so you have to enable it to work.